Assets/Xbox Live/Scripts/Social.cs

```diff
@@ -61,13 +79,16 @@ private void Start()
 
     private void Update()
     {
-        if (!string.IsNullOrEmpty(this.verticalScrollInputAxis) && Input.GetAxis(this.verticalScrollInputAxis) != 0){
+        if (!string.IsNullOrEmpty(this.verticalScrollInputAxis) && Input.GetAxis(this.verticalScrollInputAxis) != 0)
```

Comment:
Should this be wrapped in the EnableControllerInput check?

Reply:
Done. Will be in the next push.